### DISCUSSION TITLE
How to Iterate a device's interfaces and rejecting it

### DISCUSSION THREAD

**Discussion by maladi17** [pyudev/pyudev#435] [Q&A]
Hi,

Is it possible to analyze device's attributes and especially interfaces when connected (and reject a device when needed)?

Thank you

**Comment by mulkieran** [comment]
Hi! I'm not sure what you mean by interfaces.

**Comment by maladi17** [comment]
A device- say usb may have a couple of interfaces (a.k.a capabilities), for example a badusb has 2 classes like keyboard and mass storage.
